### PR TITLE
allow dropDefault at class level

### PR DESCRIPTION
**Pull request by russellremple**
@htmldoug @jducoeur @danny-goldberg-rally

### CODE REVIEW

**Review by htmldoug — APPROVED** [review]
Wonderful. Thank you so much!

**Comment by htmldoug** [comment]
Do you think it's worth a mention in the readme?

File: readme.md [hunk 1]
```diff
@@ -149,6 +156,15 @@ case class Maybe2(@dropDefault i: Option[Int] = None)
 FromScala(Maybe2(None)).transform(ToJson.string)     ==> """{}"""
 ```
 
+But `Option` types are a special case where `None` is an assumed default if a default is not provided explicitly.
```
Comment (danny-goldberg-rally) [inline readme.md:159]: could you add a test case proving this?

**Review by jducoeur — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (russellremple) [reply readme.md:159]: https://github.com/rallyhealth/weePickle/blob/63e5b5648b8b3a0bbcaa4a5a389f9aecbc13f504/weepickle/test/src/com/rallyhealth/weepickle/v1/StructTests.scala#L149

**Review by danny-goldberg-rally — APPROVED** [review]

**Review by pawanaraballi — APPROVED** [review]

**Comment by russellremple** [comment]
@htmldoug I do not have merge access to this repo

**Pull request merged**